dune-project

```diff
@@ -1 +1,2 @@
 (lang dune 1.1)
```

Comment:
we'd need to bump the dune language version to 1.7 since that's the first releast with fmt support.

reference: https://github.com/ocaml/opam-repository/pull/15560#discussion_r359332098

Reply:
I'm trying to keep the opam deps are wide as possible so we don't limit how people can use this. For `dune build @fmt`, people don't need to run that at release anyway. I wish there was a way to specify particular deps for dev but not opam :\

Reply:
I know dune doesn't fail at the moment, but it could in theory fail if it stars a check for what language version is used vs what language features we use. :\ 

Its unfortunate that the formatter can't be run via dune without this stanza in the dune-project file